### DISCUSSION TITLE
Bug in 2.0.0 for Ender3 V3SE for 0.4 nozzle

### DISCUSSION THREAD

**Discussion by wodji** [OrcaSlicer/OrcaSlicer#4828] [Dev]
In files 0.24mm Draft @Creality Ender3V3SE.json, 0.20mm Standard @Creality Ender3V3SE.json, 0.16mm Optimal @Creality Ender3V3SE.json and 0.12mm Fine @Creality Ender3V3SE.json last line of code should be: 
"compatible_printers": [
		"Creality Ender-3 V3 SE 0.4 nozzle"
	]
not:
"compatible_printers": [
		"Creality Ender-3 V3 SE"
	]

**Comment by melstoll** [comment]
Same here. We have the option to install profile for other nozzles, but all profiles for 0.4 disappear

**Comment by EndlessKustumz** [comment]
I just installed the 2.0 release as well and I have lost all my presets. I have no line widths showing, and have no quality presets that I used to have as well.
![Slicer issue](https://github.com/SoftFever/OrcaSlicer/assets/166037924/a362faa1-e38d-4396-91db-c9e4a1264ec5)

**Comment by wodji** [comment]
In folder c:\Users\user_name\AppData\Roaming\OrcaSlicer\system\Creality\process\ find files:
0.12mm Fine @creality Ender3V3SE.json
0.16mm Optimal @creality Ender3V3SE.json
0.20mm Standard @creality Ender3V3SE.json
0.24mm Draft @creality Ender3V3SE.json

and replace at bottom of those files 
from:
"compatible_printers": [
"Creality Ender-3 V3 SE"
]

to:
"compatible_printers": [
"Creality Ender-3 V3 SE 0.4 nozzle"
]

restart orca and enjoy :)

**Comment by DarkSyd007** [comment]
Thank you!  You are a lifesaver :)

**Comment by valeyard** [comment]
Just an FYI this is still broken it seems
